src/index.html

```diff
@@ -9,6 +9,61 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img 
+          src="./images/logo.png" 
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav__item">
+            <a href="#1" class="is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#2" class="is-active">
+              Samsung
+            </a>
```

Comment:
```suggestion
            <a href="#2" class="nav__link">
              Samsung
            </a>
```
only Apple link is active